tests/test_modeling_flax_common.py

```diff
@@ -67,6 +67,23 @@ def random_attention_mask(shape, rng=None):
     return attn_mask
 
 
+def floats_tensor(shape, scale=1.0, rng=None, name=None):
```

Comment:
this method already exists on master
`https://github.com/huggingface/transformers/blob/1f335aef3bb5382b5cfd7adbe5861ed4979dd98d/tests/test_modeling_flax_common.py#L63`

it should not be defined here. Please rebase this branch to pul changes from master.

Reply:
missed this eariler !!